mirage/apt/apt_inputs.py

```diff
@@ -334,7 +334,7 @@ def get_pointing_info(self, file, propid):
                             observation_label.append(obslabel)
                             observation_number.append(obsnum)
                             visit_number.append(visitnum)
-                            vid = str(propid) + visitnum + obsnum
+                            vid = str(propid) + obsnum + visitnum
                             visit_id.append(vid)
                             vgrp = '01'
```

Comment:
I am curious why the visit group and the sequence number are hard-coded, while every other parameter is either read from the pointing file or calculated. Could you maybe add a couple inline comments to denote that?

Reply:
Added some explanation. The answer is that for the sequence ID,  I'm not sure how to pull this from the xml. Specific parallel modes are mentioned in there (e.g. NIRCam-MIRI-imaging), but no dictionary of what mode relates to what number. Also, these modes were not used in APT at the time I initially wrote apt_inputs.py. For the purposes of simulating data, I figured it wasn't terribly important. For `visit groups`, I believe the values depend on how everything is scheduled/grouped in APT, and I'm not convinced it is doing that correctly at the moment with the limited information in the PPS database. Unless each 'observation' within APT is called a 'visit group' outside of APT? Something to be investigated later. Again for the purposes of creating simulated data I didn't think it was important at the time I was putting apt_inputs.py together.

Reply:
Okay, this all makes sense, thanks for clarifying! 

You're definitely right, from the perspective of generating data with MIRAGE, it doesn't really matter whether things are being "observed" in parallel or prime.

The visit group question is interesting, but also (like you say) not urgent or particularly important.